
#include "game_client_player_replied_handler.h"

///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE



void SceneClientPlayerCommonRepliedHandler::SendTipToClient(entt::entity player,const ::TipInfoMessage* request,
	::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE


}



void SceneClientPlayerCommonRepliedHandler::KickPlayer(entt::entity player,const ::GameKickPlayerRequest* request,
	::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE


}


